CMakeLists.txt

```diff
@@ -603,6 +604,9 @@ else()
 endif()
 
 if(WINDOWS)
+  if(NOT OS_LIB_DIRS)
+    set(OS_LIB_DIRS)
```

Comment:
What's going on here? Why is this being set?

Reply:
Handling OS_LIB_DIRS not defined for the removed duplicates macro

Reply:
I'm not 100% convinced that this is needed. If you remove it do builds break? I tested this on a VM and it seemed like it was building just fine. If we need it that's fine, but would you mind finding some links or more context around why this is needed? As it stands we're just adding yet another build definition, but it's not clear _who_ needs this.